api/firmware/device.go

```diff
@@ -261,7 +287,7 @@ func (device *Device) Init() error {
 
 	// Before 2.0.0, unlock was invoked automatically by the device before USB communication
 	// started.
-	if device.version.AtLeast(semver.NewSemVer(2, 0, 0)) {
+	if device.version.AtLeast(semver.NewSemVer(2, 0, 0)) && (device.status != StatusInitialized) {
```

Comment:
This is wrong I think - Initialized means backup created, not unlocked. The docstring in status.go might be wrong. Regardless it's an unrelated change.

Reply:
Yes the docstring on `StatusInitialized` is wrong, I got confused because I thought it actually meant something else in this repo (also because the animation stops playing when status changes from `StatusConnected` to `StatusInitialized`) but we actually need a new status `StatusUnlocked` and use that.

Reply:
So this can then be changed to `'initialized' || 'seeded'`:
https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/0fa7a7144984fa5a4f7731d222b69505902b220a/frontends/web/src/routes/device/bitbox02/wizard.tsx#L103

Reply:
I don't think we need StatusUnlocked. 'seeded' is the step after setting password, before creating a backup, but upon re-plug the user starts from the beginning (by design). No need to change to 'seeded' in the above link.